### ISSUE TITLE
Problem running Fudwin from TheFatRat menu 

### ISSUE THREAD

**Issue by Morsmalleo** [Screetsec/TheFatRat#592]
**PROBLEMS I AM HAVING WITH FUDWIN**

So I run setup.sh inside TheFatRat folder, everything installs fine, **I also noticed that python3 has been added for TheFatRat and python3-pip has replaced python-pip** which also installs fine, but when I go to use Powerstager from **TheFatRat menu** on the latest release of Kali Linux for Oracle Virtualbox, I get an error when creating the Fudwin. 

Fudwin.log output
`/usr/bin/x86_64-w64-mingw32-gcc`
`Traceback (most recent call last):`
  `File "tools/power.py", line 15, in <module>`
    `import urllib.parse`
`ImportError: No module named parse`

After getting the error 
`[x] There was a problem creating your FUD Rat with Powerstager` 
and after checking the Fudwin.log file in **TheFatRat/tools/** directory, I type into my terminal 
`pip3 install parse && sudo pip3 install parse`, 
the **parse** module installs without any problems, but the problem with Powerstager not working from **TheFatRat menu** still persists with the same error 
`[x] There was a problem creating your FUD Rat with Powerstager`, 
the fudwin.log file output is the same, even know I have just installed "parse" with pip3 
(I even tried `pip3 install urllib2-file` just off the top of my head to see if that would do anything and that didn't work either)

Fudwin.log (after installing the **parse** module with pip3)
`/usr/bin/x86_64-w64-mingw32-gcc`
`Traceback (most recent call last):`
  `File "tools/power.py", line 15, in <module>`
    `import urllib.parse`
`ImportError: No module named parse`

So, when that didn't work, I exited TheFatRat, and changed directories to **tools** folder, and then tried running **Powerstager** from there with 

`python3 power.py`

and **it works flawlessly**!! 

So after confirming it works manually, I tried running it from TheFatRat menu once again, but it didn't work, I got the same error

`[x] There was a problem creating your FUD Rat with Powerstager` 

and the Fudwin.log output is always the same

Fudwin.log after installing the **parse** module with **pip3** and confirming that **Powerstager** works manually from the **tools** folder 
`/usr/bin/x86_64-w64-mingw32-gcc`
`Traceback (most recent call last):`
  `File "tools/power.py", line 15, in <module>`
    `import urllib.parse`
`ImportError: No module named parse`

So in short Powerstager works manually when launched from the **tools** folder, but it does not work from **TheFatRat menu,** **AT ALL**

**CURRENT WORKAROUNDS BEING USED**
I currently have to use the following workarounds to be able to use **Powerstager** at all.
`cd TheFatRat/tools`
`python3 power.py` 

Or changing `line 15` in power.py from `urllib.parse` back to `urlparse` then installing pip2 with `wget https://bootstrap.pypa.io/2.7/get-pip.py && sudo python2.7 get-pip.py` then installing the modules "names" & "parse" with `sudo pip2 install names parse && pip2 install names parse` to be able to use Powerstager from TheFatRat menu and manually from the **tools** folder

**Comment by peterpt** [comment]
i did not had any issues installing fatrat on a clean kali VM .
Could you try on your terminal and write 'pip3 install parse' 
and see if the same issue continues ?

**Comment by Morsmalleo** [comment]
Problem fixed, I no longer need pip2 to install the names and parse modules for TheFatRat's Fudwin option to work both manually and from TheFatRat menu, cheers @peterpt you're a legend
